src/seedu/addressbook/AddressBook.java

```diff
@@ -1153,14 +1153,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, EMPTY_STRING);
+    private static String removePrefix(String fullString, String prefix) {
```

Comment:
This implementation is not abstract enough to remove only the prefix. Consider revising.